Applications/spire/include/spire/ui/flat_button.hpp

```diff
@@ -62,17 +78,23 @@ namespace spire {
       QLabel* m_label;
       mutable clicked_signal m_clicked_signal;
       bool m_clickable;
-      QString m_default_style;
-      QString m_hover_style;
-      QString m_focused_style;
-      QString m_disabled_style;
+      style m_default_style;
+      style m_hover_style;
+      style m_focus_style;
+      style m_disabled_style;
+      QString m_default_stylesheet;
```

Comment:
Remove all of the QString variants of the stylesheet. If you need them then you'll compute their values.